### ISSUE TITLE
design: activity highlighting is too subtle

### ISSUE THREAD

**Issue by peppidesu** [svsticky/radio#147]
Currently, the highlighted activity in the activities view is emphasized with a dark background:
![image](https://github.com/user-attachments/assets/82c530a9-3caf-49b4-b1c8-fa808e1f3ecf)

This can be hard to see from a distance and also looks quite dull.

I suggest we add more emphasis to the highlighted activity as well as slightly dim other activities.

**Comment by SilasPeters** [comment]
I personally do not think many others agree with this change

**Comment by SamStaijen** [comment]
> I personally do not think many others agree with this change

I stand with Silas here, In part because the poster on the right also shows the selected activity, and in part because when i fist ever saw the radio, it was clear to me what activity was selected.

**Comment by SilasPeters** [comment]
Yeah I should have explained myself as well. The highlighted activity stands out enough for me, in such a way that if we make it stand out even more it might become distracting.

Did you have any confusion @peppidesu ?
